### PR TITLE
Update to use Heroicons 2.0.13

### LINKED ISSUE #1: Update to heroicons 2.0

### PR DESCRIPTION
**Pull request by BenSpencer**
Updates heroicons-lookup to use the latest heroicons/react

This adds the following new icons:
```
AdjustmentsHorizontalIcon
AdjustmentsVerticalIcon
ArchiveBoxArrowDownIcon
ArchiveBoxIcon
ArchiveBoxXMarkIcon
ArrowDownCircleIcon
ArrowDownLeftIcon
ArrowDownOnSquareIcon
ArrowDownOnSquareStackIcon
ArrowDownRightIcon
ArrowDownTrayIcon
ArrowLeftCircleIcon
ArrowLeftOnRectangleIcon
ArrowLongDownIcon
ArrowLongLeftIcon
ArrowLongRightIcon
ArrowLongUpIcon
ArrowPathIcon
ArrowPathRoundedSquareIcon
ArrowRightCircleIcon
ArrowRightOnRectangleIcon
ArrowSmallDownIcon
ArrowSmallLeftIcon
ArrowSmallRightIcon
ArrowSmallUpIcon
ArrowTopRightOnSquareIcon
ArrowTrendingDownIcon
ArrowTrendingUpIcon
ArrowUpCircleIcon
ArrowUpLeftIcon
ArrowUpOnSquareIcon
ArrowUpOnSquareStackIcon
ArrowUpRightIcon
ArrowUpTrayIcon
ArrowUturnDownIcon
ArrowUturnLeftIcon
ArrowUturnRightIcon
ArrowUturnUpIcon
ArrowsPointingInIcon
ArrowsPointingOutIcon
ArrowsRightLeftIcon
ArrowsUpDownIcon
BackwardIcon
BanknotesIcon
Bars2Icon
Bars3BottomLeftIcon
Bars3BottomRightIcon
Bars3CenterLeftIcon
Bars3Icon
Bars4Icon
BarsArrowDownIcon
BarsArrowUpIcon
Battery0Icon
Battery100Icon
Battery50Icon
BellAlertIcon
BellSlashIcon
BellSnoozeIcon
BoltIcon
BoltSlashIcon
BookmarkSlashIcon
BookmarkSquareIcon
BugAntIcon
BuildingLibraryIcon
BuildingOffice2Icon
BuildingOfficeIcon
BuildingStorefrontIcon
CalendarDaysIcon
ChartBarSquareIcon
ChatBubbleBottomCenterIcon
ChatBubbleBottomCenterTextIcon
ChatBubbleLeftEllipsisIcon
ChatBubbleLeftIcon
ChatBubbleLeftRightIcon
ChatBubbleOvalLeftEllipsisIcon
ChatBubbleOvalLeftIcon
CheckBadgeIcon
ChevronUpDownIcon
CircleStackIcon
ClipboardDocumentCheckIcon
ClipboardDocumentIcon
ClipboardDocumentListIcon
CloudArrowDownIcon
CloudArrowUpIcon
CodeBracketIcon
CodeBracketSquareIcon
Cog6ToothIcon
Cog8ToothIcon
CommandLineIcon
ComputerDesktopIcon
CpuChipIcon
CursorArrowRaysIcon
CursorArrowRippleIcon
DevicePhoneMobileIcon
DocumentArrowDownIcon
DocumentArrowUpIcon
DocumentChartBarIcon
DocumentCheckIcon
DocumentMagnifyingGlassIcon
DocumentMinusIcon
DocumentPlusIcon
EllipsisHorizontalCircleIcon
EllipsisHorizontalIcon
EllipsisVerticalIcon
EnvelopeIcon
EnvelopeOpenIcon
ExclamationTriangleIcon
EyeDropperIcon
EyeSlashIcon
FaceFrownIcon
FaceSmileIcon
FolderArrowDownIcon
FolderMinusIcon
FolderPlusIcon
ForwardIcon
FunnelIcon
GifIcon
GiftTopIcon
GlobeAmericasIcon
GlobeAsiaAustraliaIcon
GlobeEuropeAfricaIcon
HandRaisedIcon
HandThumbDownIcon
HandThumbUpIcon
HomeModernIcon
InboxArrowDownIcon
InboxStackIcon
LanguageIcon
LifebuoyIcon
ListBulletIcon
MagnifyingGlassCircleIcon
MagnifyingGlassIcon
MagnifyingGlassMinusIcon
MagnifyingGlassPlusIcon
MapPinIcon
MegaphoneIcon
MinusSmallIcon
MusicalNoteIcon
NoSymbolIcon
PaintBrushIcon
PauseCircleIcon
PencilSquareIcon
PhoneArrowDownLeftIcon
PhoneArrowUpRightIcon
PhoneXMarkIcon
PhotoIcon
PlayCircleIcon
PlayPauseIcon
PlusSmallIcon
PowerIcon
PuzzlePieceIcon
QrCodeIcon
QueueListIcon
RadioIcon
ReceiptPercentIcon
RectangleGroupIcon
RectangleStackIcon
RocketLaunchIcon
ServerStackIcon
SignalIcon
SignalSlashIcon
SpeakerWaveIcon
SpeakerXMarkIcon
Square2StackIcon
Square3Stack3DIcon
Squares2X2Icon
SquaresPlusIcon
StopCircleIcon
SwatchIcon
TableCellsIcon
TrophyIcon
TvIcon
UserMinusIcon
UserPlusIcon
VideoCameraSlashIcon
ViewColumnsIcon
ViewfinderCircleIcon
WalletIcon
WindowIcon
WrenchIcon
WrenchScrewdriverIcon
XMarkIcon
```

And removes the following old icons (mostly renamed into the above list as far as I could see)
```
AdjustmentsIcon
AnnotationIcon
ArchiveIcon
ArrowCircleDownIcon
ArrowCircleLeftIcon
ArrowCircleRightIcon
ArrowCircleUpIcon
ArrowNarrowDownIcon
ArrowNarrowLeftIcon
ArrowNarrowRightIcon
ArrowNarrowUpIcon
ArrowSmDownIcon
ArrowSmLeftIcon
ArrowSmRightIcon
ArrowSmUpIcon
ArrowsExpandIcon
BadgeCheckIcon
BanIcon
BookmarkAltIcon
CashIcon
ChartSquareBarIcon
ChatAlt2Icon
ChatAltIcon
ChatIcon
ChipIcon
ClipboardCheckIcon
ClipboardCopyIcon
ClipboardListIcon
CloudDownloadIcon
CloudUploadIcon
CodeIcon
CollectionIcon
ColorSwatchIcon
CursorClickIcon
DatabaseIcon
DesktopComputerIcon
DeviceMobileIcon
DocumentAddIcon
DocumentDownloadIcon
DocumentRemoveIcon
DocumentReportIcon
DocumentSearchIcon
DotsCircleHorizontalIcon
DotsHorizontalIcon
DotsVerticalIcon
DownloadIcon
DuplicateIcon
EmojiHappyIcon
EmojiSadIcon
ExclamationIcon
ExternalLinkIcon
EyeOffIcon
FastForwardIcon
FilterIcon
FolderAddIcon
FolderDownloadIcon
FolderRemoveIcon
GlobeIcon
HandIcon
InboxInIcon
LibraryIcon
LightningBoltIcon
LocationMarkerIcon
LoginIcon
LogoutIcon
MailIcon
MailOpenIcon
MenuAlt1Icon
MenuAlt2Icon
MenuAlt3Icon
MenuAlt4Icon
MenuIcon
MinusSmIcon
MusicNoteIcon
OfficeBuildingIcon
PencilAltIcon
PhoneIncomingIcon
PhoneMissedCallIcon
PhoneOutgoingIcon
PhotographIcon
PlusSmIcon
PuzzleIcon
QrcodeIcon
ReceiptTaxIcon
RefreshIcon
ReplyIcon
RewindIcon
SaveAsIcon
SaveIcon
SearchCircleIcon
SearchIcon
SelectorIcon
SortAscendingIcon
SortDescendingIcon
SpeakerphoneIcon
StatusOfflineIcon
StatusOnlineIcon
SupportIcon
SwitchHorizontalIcon
SwitchVerticalIcon
TableIcon
TemplateIcon
TerminalIcon
ThumbDownIcon
ThumbUpIcon
TranslateIcon
TrendingDownIcon
TrendingUpIcon
UploadIcon
UserAddIcon
UserRemoveIcon
ViewBoardsIcon
ViewGridAddIcon
ViewGridIcon
ViewListIcon
VolumeOffIcon
VolumeUpIcon
XIcon
ZoomInIcon
ZoomOutIcon
```

Closes #1

### CODE REVIEW

File: src/outlineIcons.ts [hunk 1]
```diff
@@ -1,49 +1,103 @@
-import * as Icons from "@heroicons/react/outline"
+//import * as Icons from "@heroicons/react/outline"
```
Comment (naquiroz) [inline src/outlineIcons.ts]:
```suggestion
```

**Review by naquiroz — CHANGES_REQUESTED** [review]
Left a small change.

File: package-lock.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "heroicons-lookup",
-  "version": "1.1.0",
+  "version": "1.3.2",
```
Comment (naquiroz) [inline package-lock.json]:
```suggestion
  "version": "2.0.0",
```
Remember to change the version here too.

File: src/outlineIcons.ts (hunk 1 above)
Comment (BenSpencer) [reply src/outlineIcons.ts]: Not sure where that came from. Thanks!

File: src/outlineIcons.ts [hunk 3]
```diff
@@ -1,49 +1,102 @@
-import * as Icons from "@heroicons/react/outline"
+import * as Icons from "@heroicons/react/24/outline"
```
Comment (naquiroz) [inline src/outlineIcons.ts:1]: hmm, I'm thinking that the size might be another option add, meaning, it should be asked on the lookup function. What do you think?

**Review by naquiroz — CHANGES_REQUESTED** [review]

File: src/outlineIcons.ts (hunk 3 above)
Comment (BenSpencer) [reply src/outlineIcons.ts:1]: I think the only options they have are 24/outline, 24/solid and "mini" which is 20/solid. So maybe we could just add a third option to `IconFormat`. What do you think?

```
export type IconFormat = "solid" | "outline" | "mini"
```

Comment (naquiroz) [reply src/outlineIcons.ts:1]: Oh, if that's the case, then yes, sounds good.

Comment (BenSpencer) [reply src/outlineIcons.ts:1]: Nice. I've added that as well then. 👍

**Review by naquiroz — CHANGES_REQUESTED** [review]
One small last request, please add the "mini" option in the docs on the README.md 🙏🏻

**Comment by BenSpencer** [comment]
Good catch. Added to the readme as well. I also couldn't find any other references to the previous styles which were not being handled for the new one.

**Review by naquiroz — APPROVED** [review]

**Comment by naquiroz** [comment]
Great work!

**Comment by naquiroz** [comment]
Will push this release later in the day, haven't done any GHA yet for continuous deployment.

**Comment by Sparticuz** [comment]
I don't think this has been published to npm?

**Comment by naquiroz** [comment]
It's not, I ll do it this month, apologies.

**Pull request merged**